treenode.go

```diff
@@ -286,7 +286,11 @@ func (n *TreeNodeInstance) closeDispatch() error {
 		n.msgDispatchQueueWait <- true
 	}
 	n.msgDispatchQueueMutex.Unlock()
-	return n.ProtocolInstance().Shutdown()
+	pni := n.ProtocolInstance()
+	if pni == nil {
```

Comment:
was in a test where the system started to set up and then has been teared down before it was properly set up.